src/bl4ckscor3/plugin/animalessentials/cmd/Spawn.java

```diff
@@ -359,7 +353,7 @@ else if(slot == 2) //setting type
 				openRabbitType(p);
 			else if(slot == 7) //spawning
 			{
-				CraftRabbit r = (CraftRabbit)p.getWorld().spawnEntity(p.getLocation(), EntityType.RABBIT);
+				Rabbit r = (Rabbit)p.getWorld().spawnEntity(p.getLocation(), EntityType.valueOf("RABBIT"));
```

Comment:
Hey, thanks for your contribution! I really appreciate it that people like the project and are willing to help :)
One question for my understanding: Why is this the only part where you use valueOf() instead of the direct reference?